worth2/templates/main/journal_base.html

```diff
@@ -1,6 +1,6 @@
 {% extends 'base.html' %}
 
-{% block title %}Journal for Session {{ session_num }}{% endblock %}
+{% block title %}Journal for Session {{ session_num }} | Participant: {{ participant.study_id }}{% endblock %}
```

Comment:
I had thought the study_id was strictly internal? What's the reasoning behind exposing it directly to the research participant?